src/xcode/ENA/ENA/Source/Scenes/Checkin/EditCheckInTime/Cells/CheckInTimeCell.swift

```diff
@@ -76,6 +73,10 @@ class CheckInTimeCell: UITableViewCell, ReuseIdentifierProviding {
 		dateTimeLabel.textAlignment = .right
 		dateTimeLabel.numberOfLines = 1
 
+		// we add a placeholder text to get the cell hight calculation right
+		// otherwise the label is hidden, text gets updated as soon as configure with a cellViewModel gets called
+		dateTimeLabel.text = "placeholder"
```

Comment:
Maybe this can be avoided by adjusting the settings of the stack view? Did you try to adjust `stackView.distribution` to a different value?